lab_6_pipeline/pipeline.py

```diff
@@ -128,11 +218,18 @@ def convert_pos(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Extracts and converts POS from the OpenCorpora tags into the UD format
         """
+        return self._tag_mapping[self.pos][tags.POS or 'UNKN']
 
     def convert_morphological_tags(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Converts the OpenCorpora tags into the UD format
         """
+        ud_tags = {}
+        for category in ( 'animacy', 'case', 'gender', 'number'):
+            if not (open_corpora_tag := eval(f'tags.{category}')):
+                continue
+            ud_tags[eval(f'self.{category}')] = eval(f"self._tag_mapping[self.{category}]['{open_corpora_tag}']")
```

Comment:
come-on, lets say I have not seen so much evals in one place

Reply:
fixed)